### ISSUE TITLE
Shutdown button does not work

### ISSUE THREAD

**Issue by Jeffiy** [jupyter/notebook#563]
when I clicked the shutdown button, I got a 404 issue like this:
`
[W 10:15:09.387 NotebookApp] 404 DELETE /api/sessions/%5Bobject%20Object%5D (127.0.0.1) 1.41ms referer=https://localhost:8888/tree
`

and another 404 was 
`
[W 10:17:48.998 NotebookApp] 404 GET /nbextensions/widgets/notebook/js/extension.js (127.0.0.1) 1.52ms referer=https://localhost:8888/notebooks/test.ipynb
`

**Comment by minrk** [comment]
@Jeffiy what version of the notebook?

**Comment by Jeffiy** [comment]
pip list
```
decorator (4.0.4)
ipykernel (4.1.0)
ipyparallel (4.0.2)
ipython (4.0.0)
ipython-genutils (0.1.0)
ipywidgets (4.0.3)
Jinja2 (2.8)
jsonschema (2.5.1)
jupyter-client (4.0.0)
jupyter-core (4.0.6)
MarkupSafe (0.23)
mistune (0.7.1)
nbconvert (4.0.0)
nbformat (4.0.1)
notebook (4.1.0.dev0, /home/jeffiy/Documents/py/notebook)
path.py (8.1.2)
pexpect (4.0.1)
pickleshare (0.5)
pip (7.1.2)
ptyprocess (0.5)
Pygments (2.0.2)
pyzmq (14.7.0)
setuptools (18.2)
simplegeneric (0.8.1)
terminado (0.5)
tornado (4.2.1)
traitlets (4.0.0)
wheel (0.24.0)
```
and python3.5

**Comment by minrk** [comment]
Since this is a dev install, can you verify that you are on latest master and run `python setup.py js css`?

**Comment by minrk** [comment]
@Jeffiy should be fixed in master by #579
